packages/odie-client/src/hooks/use-get-most-recent-open-conversation.tsx

```diff
@@ -52,9 +52,9 @@ export const useGetMostRecentOpenConversation = () => {
 		} );
 
 		if ( sortedConversations?.length > 0 ) {
-			supportInteractionId = sortedConversations[ 0 ]?.metadata?.supportInteractionId;
-			userHasRecentOpenConversation = true;
+			mostRecentSupportInteractionId = sortedConversations[ 0 ]?.metadata?.supportInteractionId;
+			totalNumberOfConversations = sortedConversations.length;
 		}
 	}
-	return { userHasRecentOpenConversation, supportInteractionId };
+	return { mostRecentSupportInteractionId, totalNumberOfConversations };
```

Comment:
Would it work if we just return

`return {
	mostRecentSupportInteractionId: sortedConversations?.[ 0 ]?.metadata?.supportInteractionId;
	totalNumberOfConversations: sortedConversations?.length
}
`

Reply:
Let's see. I'll try that.